### DISCUSSION TITLE
Allow multiple purchase as combination with quantity relation

### DISCUSSION THREAD

**Discussion by ballashop** [PrestaShop/PrestaShop#40176] [Ideas]
#### Is your feature request related to a problem?

Hi, at the moment it is not possible to sell multiple products in the same product schede on PrestaShop.
At the most it is possible to indicate a minimum product purchase but not a multiplex of them.
I'll explain better:
I can create an attribute named Quantity to use as a combination:
<img width="306" alt="Schermata 2021-03-12 alle 15 25 16" src="https://user-images.githubusercontent.com/28685316/110953231-67a95780-8347-11eb-9e76-be9afc272bf9.png">
single, 2 pc, 6pc, 12pc etc ...

I associate this attribute with the apple product, so I have the possibility to sell the apple both individually and in multiples decided by me.
<img width="969" alt="Schermata 2021-03-12 alle 15 25 10" src="https://user-images.githubusercontent.com/28685316/110953309-7c85eb00-8347-11eb-97ce-cd940c25b716.png">

here's how it would look on the site:
<img width="1191" alt="Schermata 2021-03-12 alle 15 25 42" src="https://user-images.githubusercontent.com/28685316/110953440-a63f1200-8347-11eb-8033-252d34e6aa29.png">

In this way, however, I have no relation of quantity between the combinations,
if I have 100 apples available, I should divide these 100 apples on each combination, like this:
single -> 24
2pc -> 10 (20)
6pc -> 3 pcs (18)
12pc -> 1pc (12)
26pc -> 1pc (26)
= 100
the system works if I enter the stock, manually. But what happens if the stock is loaded by an external tool connected to PrestaShop?
We imagine that arrive 1000 pieces of apples every day, how do I update the stock by hand?
Also, if I have a physical store of apples, and I sell a lot of them a day, the stock has to constantly update, but how can I do if I have these combinations without a relationship between them?
One important thing is missing, I believe, within the options of each combination:
there is the **Impact on weight**, but the impact on the **Package dimension** is missing
![Schermata-2021-03-12-alle-15 49 56](https://user-images.githubusercontent.com/28685316/110957109-9aede580-834b-11eb-969d-9095809a1319.jpg)
I hope I have explained well.
Thanks a lot.
Balla

**Comment by florine2623** [comment]
Hello @ballashop ,

I will ping the product team regarding your first suggestion.
Regarding the impact on Package Dimension, please open a new issue about it. These are 2 separate features.

Thanks

**Comment by florine2623** [comment]
Ping @PrestaShop/product-team ,

What do you think about this ?
At the moment, linking stock quantity in combination on product sheet isn't possible through Attributes nor Specific Prices.

Thanks!
